pom.xml

```diff
@@ -179,6 +179,8 @@
     <commons-lang2.version>2.6</commons-lang2.version>
     <!-- org.apache.commons/commons-lang3/-->
     <commons-lang3.version>3.8.1</commons-lang3.version>
+    <!-- org.apache.commons/commons-pool2/-->
+    <commons-pool2.version>2.6.0</commons-pool2.version>
```

Comment:
There's a newer one now, probably should use it.

Reply:
Will check the recent version and reflect it.